ens/ens.py

```diff
@@ -65,8 +67,9 @@
 
 if TYPE_CHECKING:
     from web3 import Web3  # noqa: F401
-    from web3.contract import (  # noqa: F401
+    from web3.contract.contract import (  # noqa: F401
         Contract,
+        ContractFunction,
```

Comment:
I don't see any real utility in adding `ContractFunction` to the init. I don't think anyone would build their own `ContractFunction` object and it would just add to the list that is imported there so we can point to it easier internally. Thoughts?